Lib/glyphsLib/builder/glyphs.py

```diff
@@ -12,8 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from glyphsLib import classes
 from glyphsLib import builder
```

Comment:
get rid of the circular import here, instead import the `GLYPHS_PREFIX` that you need below directly from the `ufo` module where it's defined.

Actually these symbols that are shared by both `ufo` and `glyphs` could be defined in the builder/__init__.py, or in a `constants` or similar sub-module of builder package.